org.darktable.Darktable.json

```diff
@@ -78,8 +78,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "http://www.exiv2.org/exiv2-0.25.tar.gz",
-                    "sha256": "c80bfc778a15fdb06f71265db2c3d49d8493c382e516cb99b8c9f9cbde36efa4"
+                    "url": "http://www.exiv2.org/builds/exiv2-0.26-trunk.tar.gz",
```

Comment:
Seems like a mistake, these appear to be binary builds.

Reply:
Yes, for some reason version 0.26 has that "-trunk" appendix which earlier version didn't seem to have.